### DISCUSSION TITLE
Custom Inventory Report

### DISCUSSION THREAD

**Discussion by farmaspace** [saleweaver/python-amazon-sp-api#93] [Q&A]
Hi, 

I'm trying to get a custom report with the following command, but a non-customized report of all offers is still created.

rep = Reports(marketplace=Marketplaces.IT, account='default').create_report(reportOptions='custom=true', reportType=ReportType.GET_MERCHANT_LISTINGS_ALL_DATA.value)

Can anyone help me?

Thanks in advance

**Comment by saleweaver** [comment]
Please try with `reportOptions={'custom‘: True}`

Am 11. März 2021, 10:11 +0100 schrieb farmaspace ***@***.***>:


Hi,

I'm trying to get a custom report with the following command, but a non-customized report of all offers is still created.

rep = Reports(marketplace=Marketplaces.IT, account='default').create_report(reportOptions='custom=true', reportType=ReportType.GET_MERCHANT_LISTINGS_ALL_DATA.value)

Can anyone help me?

Thanks in advance

**Reply by farmaspace** [reply]
Thank you very much. Now it works perfectly
